config/_ytt_lib/github.com/cloudfoundry/capi-k8s-release/values.yml

```diff
@@ -21,6 +22,11 @@ ccdb:
   database: cloud_controller
 
 blobstore:
+  region: "''"
```

Comment:
is this actually `''` as a value of region string?

Reply:
This doesn't currently cause any problems so we are not worrying about it before we merge this PR. We will follow up with CAPI afterwards.